### DISCUSSION TITLE
[feature] Add user setting for file extension configuration

### DISCUSSION THREAD

**Discussion by jaketig** [nuxtrdev/nuxtr-vscode#118] [Ideas]
We use .js files instead of .ts files for plugins/apis/composables/stores/etc... 

This appears to be hardcoded here: https://github.com/nuxtrdev/nuxtr-vscode/blob/main/src/utils/commands.ts#L3

Any chance this could be made configurable?

**Comment by zhaolinlau** [comment]
Hi @jaketig , I have opened a PR #159
